.github/workflows/comparison-results.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
       # langauge specific repository file
       - name: Run the script for ${{github.head_ref}} and ${{github.base_ref}}
-        run: export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} && cd ./temp/standalone-monitoring-stability && pip install -r requirements.txt && python3 ./run.py -r ./repos/${{matrix.language}}.txt -b ${{github.base_ref}} -h ${{github.head_ref}} -bcr ${{ github.event.pull_request.base.repo.html_url }} -hcr ${{ github.event.pull_request.head.repo.html_url }} -guf
+        run: export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }} && export SLACK_TOKEN=${{ secrets.SLACK_TOKEN }} && cd ./temp/standalone-monitoring-stability && pip install -r requirements.txt && python3 ./run.py -r ./repos/${{matrix.language}}.txt -b ${{github.base_ref}} -h ${{github.head_ref}} -bcr ${{ github.event.pull_request.base.repo.html_url }} -hcr ${{ github.event.pull_request.head.repo.html_url }} -guf
```

Comment:
Can we use the `env` key for this? It is much cleaner, and will help if we want to add more environment variables.

Reply:
Yes right, need to change at multiple places like where its fetched on comparison repo.

Reply:
Yes, discovered this while working on the pipeline. Should be a small change.